src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -107,21 +108,23 @@ export class ListOfComplaintsComponent implements OnInit {
         const studentParticipation = complaint.result.participation as StudentParticipation;
         const exercise = studentParticipation.exercise;
         const submissionId = complaint.result.submission.id;
-
         if (!exercise || !exercise.type || !submissionId) {
             return;
         }
+        if (!this.correctionRound) {
+            this.correctionRound = 0;
+        }
```

Comment:
```suggestion
        this.correctionRound ?? 0;
```

Reply:
I guess that would be more concise, but also less readable so I'll keep it. :)